### DISCUSSION TITLE
Support for IntelliDock?

### DISCUSSION THREAD

**Discussion by bluppus** [rxhanson/Rectangle#581] [Ideas]
This app is superior to the similar app "Grid" in every way, except that there is no support for IntelliDock. The issue is shown on its website ( https://mightymac.app/intellidock/ ) and I would be very thankful if an option for this was added.
Put into words, Rectangle should (if the option is enabled) expand apps into the space of the dock even if it is currently visible, since Intellidock will then hide the dock automatically. Right now, for example maximizing an app with rectangle leaves an unused area where the dock was. 

![image](https://user-images.githubusercontent.com/59601244/134815909-944dd887-7803-468a-9cc5-670be2589d09.png)

**Comment by rxhanson** [comment]
I don’t currently have plans to add in special cased behavior for IntelliDock but would consider a PR for it.

**Reply by null-von-sushi** [reply]
I am guessing this is the case even for Rectangle Pro?
Asking because I just got a Mac and I do not have the ability to implement this unfortunately

**Reply by rxhanson** [reply]
This is the case for Rectangle Pro as well, although I may consider adding this to Rectangle Pro in a future update.
